osu.Game/Screens/Ranking/Expanded/Accuracy/AccuracyCircle.cs

```diff
@@ -74,12 +74,12 @@ public partial class AccuracyCircle : CompositeDrawable
         /// <summary>
         /// SS is displayed as a 1% region, otherwise it would be invisible.
         /// </summary>
-        private const double virtual_ss_percentage = 0.01;
+        public const double VIRTUAL_SS_PERCENTAGE = 0.01;
 
         /// <summary>
-        /// The width of a <see cref="RankNotch"/> in terms of accuracy.
+        /// The width of spacing in terms of accuracy between the grade circles.
         /// </summary>
-        public const double NOTCH_WIDTH_PERCENTAGE = 1.0 / 360;
+        public const double GRADE_SPACING_PERCENTAGE = 2.0 / 360;
```

Comment:
Why has this increased two-fold too?

Reply:
It's likely going to be for the same reason as https://github.com/ppy/osu/pull/26769#discussion_r1469654687.

Reply:
This one is different. Previously we were drawing boxes with `EdgeSmoothness = 1` on top of a circle, which resulted in visible spacing being approx `const + 2px`. Now it's the opposite: antialiasing is being handled by the circles and the visible spacing would be `const - 2px`. And to account for that I increased the const